.openapi-generator/FILES

```diff
@@ -220,28 +256,17 @@ docs/TimeOffEndpointRequest.md
 docs/TimeOffRequest.md
 docs/TimeOffResponse.md
 docs/TimeOffStatusEnum.md
-docs/TypeEnum.md
 docs/UnitsEnum.md
 docs/ValidationProblemSource.md
 docs/WarningValidationProblem.md
+docs/WebhookReceiver.md
+docs/WebhookReceiverRequest.md
+docs/WebhookReceiversApi.md
 git_push.sh
 requirements.txt
 setup.cfg
 setup.py
 test-requirements.txt
 test/__init__.py
-test/test_account_type_enum.py
```

Comment:
Uh oh, @leewang0 @dan-merge have y'all seen OpenAPI deleting lines from this file like this before?

Reply:
Ah, this requires some context: at one point we had some endpoints for hr benefits info that were renamed. However, we kept some of the old flavor in some sdk's because they were used by a single customer. Please ping Amanda P. to see if that's still the case for these time_off_balances etc api's, and if not feel free to remove.

And depending on what we promised them, it may be safe to remove them anyway if we've given enough warning (it's been months now i think).

Worst case, you restore any deleted files, keeping the deprecated endpoints in this sdk.

Reply:
Ah yeah, deprecated APIs are still in use right now so I'll keep these around